### PR TITLE
adds unit tests for Organization model

### LINKED ISSUE #526: Write unit tests for the Organization model
Write unit tests for the [`Organization` Model](https://github.com/code-corps/code-corps-ember/blob/develop/app/models/organization.js). There is an [existing unit test file](https://github.com/code-corps/code-corps-ember/blob/develop/tests/unit/models/organization-test.js), but it could use more coverage (computed properties & relationship declarations). 

## References
- Progress on: #98 
- [Model testing guide](https://guides.emberjs.com/v2.8.0/testing/testing-models/)
- [Contributing Guidelines](https://github.com/code-corps/code-corps-ember/blob/develop/.github/CONTRIBUTING.md)
- Join our [Slack](https://codecorps.slack.com)

### PR DESCRIPTION
**Pull request by idelairre**
# What's in this PR?

Organization model tests for attributes, relationships and computed properties.

## References
Fixes #526

Progress on: #98

### CODE REVIEW

File: tests/unit/models/organization-test.js [hunk 1]
```diff
@@ -1,12 +1,63 @@
...
+
+  assert.equal(relationship.key, 'projects', 'has relationship with organization');
+  assert.equal(relationship.kind, 'hasMany', 'kind of relationship is hasMany');
+});
+
+test('should have computer properties for pending memberships', function(assert) {
+  assert.expect(3);
+
+  let organization;
+  const members = [];
```
Comment (JoshSmith) [inline tests/unit/models/organization-test.js]: Not sure if `const` is right here?

**Comment by JoshSmith** [comment]
@idelairre looks like there are some conflicts with #589, i.e. @filipecrosk got to this first.

**Comment by JoshSmith** [comment]
This is kind of a bummer of a collision. Sorry for the confusion here. 😞

**Comment by idelairre** [comment]
No problem, I'll troll the open issues and see what else I can work on.

**Comment by JoshSmith** [comment]
Thanks @idelairre. Feel free to stop by our Slack too and ping me for ideas!

**Pull request closed without merging**